File: app/src/main/java/dev/maxsiomin/domainsearch/repository/domainrepository/Status.java
package dev.maxsiomin.domainsearch.repository.domainrepository;

/**
 * Status for search history
 */
public enum Status {

    /**
     * Retrofit error
     */
    FAILURE,

    /**
     * Domain not found
     */
    NOT_FOUND,

    /**
     * Domain found
     */
    FOUND,
}

File: app/src/main/java/dev/maxsiomin/domainsearch/repository/domainrepository/Status.kt
package dev.maxsiomin.domainsearch.repository.domainrepository

/**
 * Status for search history
 */
enum class Status {

    /**
     * Retrofit error
     */
    FAILURE,

    /**
     * Domain not found
     */
    NOT_FOUND,

    /**
     * Domain found
     */
    FOUND,
}
